homeassistant/components/onewire/strings.json

```diff
@@ -30,7 +30,8 @@
     },
     "sensor": {
       "counter_id": {
-        "name": "Counter {id}"
+        "name": "Counter {id}",
+        "unit_of_measurement": "count"
```

Comment:
Hmmm.... I'm not sure this one is correct.
count is not a unit, and it feels like this should be dropped altogether instead.

(note, the same probably applies to `rfxtrx`)

Reply:
I thought about that, too (and how to translate it in a meaningful way).